### ISSUE TITLE
[in synopsysctl] Clarify which inputs are absolutely required for deploying Polaris

### ISSUE THREAD

**Issue by yashbhutwala** [blackducksoftware/synopsys-operator#1067]
**What would you like to be added**:
Make it clear to the "deployer of Polaris" which fields are required to deploy. This includes conditional requirements, for example if postgres container is used, only postgres-password is required. if postgres container is not used, postgres-host, postgres-port, postgres-username, and postgres-password are required.

**Why is this needed**:
Better user experience.

**Comment by mphammer** [comment]
One way we could start to do this is by adding more examples to the --help that show the flags that go together.

**Comment by yashbhutwala** [comment]
Good idea! 😄 Added an example in #1107.  While better than nothing, this is still just a workaround, need to find a better way to build our own template with `cobra` similar to `kubectl` to group options together and mark group requirements 😄
